### PR TITLE
Fixed Missing Notebook links

### PR DESCRIPTION
**Pull request by leestott**
Fixing Lesson 7 Readme file which was missing links to OpenAI and Azure OpenAI notebooks see https://github.com/microsoft/generative-ai-for-beginners/issues/155

Fix 
You can also try it out on your own in the following [notebook for OpenAI](./notebook-openai.ipynb?WT.mc_id=academic-105485-koreyst) or [notebook for Azure OpenAI Services](./notebook-azure-openai.ipynb?WT.mc_id=academic-105485-koreys) for this lesson.

### CODE REVIEW

**Comment by leestott** [comment]
> We have automatically detected the following broken relative paths in your lessons. Please check the file paths and associated broken paths inside them. Learn more from our [contributing guide](https://github.com/microsoft/generative-ai-for-beginners/blob/main/CONTRIBUTING.md).
> 
> # Check Broken Paths
> ```
>   FILE './07-building-chat-applications/translations/cn/README.md'
> has the following broken relative paths ['./notebook-openai.ipynb', './notebook-azure-openai.ipynb']
> ```

These are broken links as its using the English notebooks so this expected.

File: 07-building-chat-applications/translations/cn/README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@
 - **更容易维护**：更新和改进更易于管理，因为大多数 API 和 SDK 在发布新版本时只需要更新库。
 - **获得尖端技术**：利用经过微调和在广泛数据集上训练的模型为您的应用程序提供自然语言功能。
 
-访问 SDK 或 API 的功能通常涉及获取使用所提供服务的许可，这通常是通过使用唯一 kwy 或身份验证 token 来实现的。 我们将使用 OpenAI Python library 来探索它是什么样子。 您也可以在本章的[notebook](./notebook.ipynb?WT.mc_id=academic-105485-koreyst) 中自行尝试。
+访问 SDK 或 API 的功能通常涉及获取使用所提供服务的许可，这通常是通过使用唯一 kwy 或身份验证 token 来实现的。 我们将使用 OpenAI Python library 来探索它是什么样子。 您也可以在本章的[OpenAI notebook](./notebook-openai.ipynb?WT.mc_id=academic-105485-koreyst) [Azure OpenAI notebook](./notebook-azure-openai.ipynb?WT.mc_id=academic-105485-koreyst)  中自行尝试。
```
Comment (john0isaac) [inline 07-building-chat-applications/translations/cn/README.md:65]:
```suggestion
访问 SDK 或 API 的功能通常涉及获取使用所提供服务的许可，这通常是通过使用唯一 kwy 或身份验证 token 来实现的。 我们将使用 OpenAI Python library 来探索它是什么样子。 您也可以在本章的[OpenAI notebook](../../notebook-openai.ipynb?WT.mc_id=academic-105485-koreyst) [Azure OpenAI notebook](../../notebook-azure-openai.ipynb?WT.mc_id=academic-105485-koreyst)  中自行尝试。
```

Comment (john0isaac) [reply 07-building-chat-applications/translations/cn/README.md:65]: the link is not relatively right the notebook is located two parent directories away from where the Chinese readme is located.
using the first `../` moves you to the translations directory then using another `../` will move you to the directory where the notebook is located and you can access its name.

Comment (leestott) [reply 07-building-chat-applications/translations/cn/README.md:65]: @john0isaac thanks for checking. I will make the changes

**Review by softchris — APPROVED** [review]

**Pull request merged**